packages/boot-contract-derive/Cargo.toml

```diff
@@ -11,5 +11,11 @@ description = "Attribute macro for creating a contract interface."
 proc-macro = true
 
 [dependencies]
+convert_case = "0.6.0"
+cosmwasm-std.workspace = true
```

Comment:
there is actually no need to import these as a macro works on the AST level (before any of these crates are interpreted).

Reply:
Done in https://github.com/Abstract-OS/BOOT/pull/77/commits/f4648aea95d6901898a9330a4b579ac25fb3be61